packs/travisci/config.yaml

```diff
@@ -0,0 +1,7 @@
+git_auth_key: Your Git Authentication Key
```

Comment:
We should simplify the config - only thing which user really needs to specify is the Travis Auth token.